examples/dynatrace-express-apm/src/sdk-v3/sdk.js

```diff
@@ -1,46 +1,44 @@
-const path = require('path')
-const { ClientBuilder } = require('@commercetools/sdk-client-v2')
-// const { createTelemetryMiddleware } = require('@commercetools/ts-sdk-apm')
-const {
-  createTelemetryMiddleware,
-} = require('@commercetools/ts-sdk-apm/dist/commercetools-ts-sdk-apm.cjs.js')
+const { ClientBuilder } = require('@commercetools/ts-client')
+const { createTelemetryMiddleware } = require('@commercetools/ts-sdk-apm')
 const { createApiBuilderFromCtpClient } = require('@commercetools/platform-sdk')
 const fetch = require('node-fetch')
+const path = require('path')
 
 const oneagent = path.join(__dirname, '..', '..', 'oneagent-tracer.js')
+const tracer = require(oneagent)
+
 const projectKey = process.env.CTP_PROJECT_KEY
 const authMiddlewareOptions = {
-  host: 'https://auth.europe-west1.gcp.commercetools.com',
+  host: process.env.CTP_AUTH_URL,
   projectKey,
   credentials: {
     clientId: process.env.CTP_CLIENT_ID,
     clientSecret: process.env.CTP_CLIENT_SECRET,
-    user: {
-      username: process.env.CTP_CLIENT_USERNAME,
-      password: process.env.CTP_CLIENT_PASSWORD,
-    },
   },
   scopes: [`manage_project:${projectKey}`],
-  fetch,
+  httpClient: fetch,
 }
 
 const httpMiddlewareOptions = {
-  host: 'https://api.europe-west1.gcp.commercetools.com',
+  host: process.env.CTP_API_URL,
```

Comment:
Please update the `.env.example` file to include this new variable

Reply:
you are right, I forgot to update it

Reply:
We have an [endpoint](https://github.com/commercetools/commercetools-sdk-typescript/blob/62fb08d43dbcd7713cf8e49864a511a15a8b2299/examples/dynatrace-express-apm/src/service/CartService.js#L28) that required this, that is why we used the password flow originally, except there is a way to do this without the password flow, else this endpoint will fail.

I will still maintain we revert this change back to use the password flow.